psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -2684,10 +2684,7 @@ std::vector<SharedMatrix> MintsHelper::ao_overlap_kinetic_deriv2_helper(const st
 
 /* 1st derivatives of electric dipole integrals in the AO basis */
 std::vector<SharedMatrix> MintsHelper::ao_elec_dip_deriv1_helper(int atom) {
-    std::vector<std::string> cartcomp;
-    cartcomp.push_back("X");
-    cartcomp.push_back("Y");
-    cartcomp.push_back("Z");
+    std::vector<std::string> cartcomp{ "X", "Y", "Z" };
```

Comment:
@loriab Is this the change that you were hoping for? Or something else?

Reply:
Line 3208 was where I had in mind. Did you get the cmake change? That’s the vital one before merge. I wouldn’t go all out on mintsgelper — just initializer list the xyz occurrences in your code.

Reply:
Line 3208 was where I had in mind. Did you get the cmake change? That’s the vital one before merge. I wouldn’t go all out on mintsgelper — just initializer list the xyz occurrences in your code.

Reply:
Ah, yes. My mistake. I made the change for my `ao_elec_dip_deriv1_helper` first, but missed the `mo_elec_dip_deriv1` function. I fixed that one as well now. And added the line with my test to CMakeLists.txt as well. Thanks.